project/Dependencies.scala

```diff
@@ -48,7 +48,8 @@ object Dependencies {
   lazy val test = Seq(
     "junit" % "junit" % "4.13.2" % "it,test",
     "com.novocode" % "junit-interface" % "0.11" % "it,test",
-    "org.scalacheck" %% "scalacheck" % "1.15.4" % "it,test"
+    "org.scalacheck" %% "scalacheck" % "1.15.4" % "it,test",
+    "xmlunit" % "xmlunit" % "1.6" % "test"
```

Comment:
Did you verify the license of XML unit and any dependecies? I also don't see it used any where? Can this be removed?

Reply:
Will remove or verify license. I like that there is an official XML such tool rather than rolling our own. That said, turns out I didnt need this.

Reply:
removed.